Modules/Filtering/ImageFilterBase/wrapping/itkBinaryGeneratorImageFilter.wrap

```diff
@@ -24,4 +24,18 @@ itk_wrap_class("itk::BinaryGeneratorImageFilter" POINTER)
   if(ITK_WRAP_rgb_unsigned_char AND ITK_WRAP_unsigned_char)
     itk_wrap_image_filter_combinations(UC "${label_types}" RGBUC)
   endif()
+
+  message("RGB(A) BinaryGeneratorImageFilter wrapping")
+  if(ITK_WRAP_rgb_unsigned_char)
+    itk_wrap_image_filter(RGBUC 3)
+  endif()
+  if(ITK_WRAP_rgb_unsigned_short)
+    itk_wrap_image_filter(RGBUS 3)
+  endif()
+  if(ITK_WRAP_rgba_unsigned_char)
+    itk_wrap_image_filter(RGBAUC 3)
+  endif()
+  if(ITK_WRAP_rgba_unsigned_short)
+    itk_wrap_image_filter(RGBAUS 3)
+  endif()
```

Comment:
We could replace these 12 lines with the single line:
```
itk_wrap_image_filter("${WRAP_ITK_COLOR}" 3)
```

Reply:
@Leengit good suggestion.

@dzenanz looks good, but we also want to use this for `itkAddImageFilter.wrap`.